File: Data Structures & Algorithms/number-of-sub-arrays-of-size-k-and-average-greater-than-or-equal-to-threshold/submission-0.cpp
class Solution {
public:
    int numOfSubarrays(vector<int>& arr, int k, int threshold) {
        int sum = 0;
        int i;
        for (i=0; i<k; i++){
            sum+=arr[i];
        }
        int count = 0;
        if (sum/k>=threshold) count++;
        while (i<arr.size()){
            sum+=arr[i];
            sum-=arr[i-k];
            cout << sum;
            if (sum/k>=threshold) count++;
            i++;
        }
        return count;
    }
};

File: Data Structures & Algorithms/palindromic-substrings/submission-0.py
class Solution:
    def expand(self, s, l, r):
        if s[l]!=s[r]:
            return 0
        count = 1
        while (l>0 and r<len(s)-1):
            if s[l-1]==s[r+1]:
                l-=1
                r+=1
                count+=1
            else:
                break
        return count
    def countSubstrings(self, s: str) -> int:
        count = 0
        for i in range(len(s)):
            count+=self.expand(s,i,i)
        for i in range(len(s)-1):
            count+=self.expand(s,i,i+1)
        return count